packages/app/src/DBSearchPage.tsx

```diff
@@ -1003,17 +1004,36 @@ function DBSearchPage() {
                 setNewSourceModalOpened(true);
               }}
             />
-            <ActionIcon
-              variant="subtle"
-              color="dark.2"
-              size="sm"
-              onClick={() => setModelFormExpanded(v => !v)}
-              title="Edit Source"
-            >
-              <Text size="xs">
-                <i className="bi bi-gear" />
-              </Text>
-            </ActionIcon>
+            <Menu withArrow position="bottom-start">
+              <Menu.Target>
+                <ActionIcon
+                  variant="subtle"
+                  color="dark.2"
+                  size="sm"
+                  title="Edit Source"
+                >
+                  <Text size="xs">
+                    <i className="bi bi-gear" />
+                  </Text>
+                </ActionIcon>
+              </Menu.Target>
+              <Menu.Dropdown>
+                <Menu.Label>Sources</Menu.Label>
+                <Menu.Item
+                  leftSection={<i className="bi bi-plus-circle" />}
+                  onClick={() => setNewSourceModalOpened(true)}
+                >
+                  Create New Source
+                </Menu.Item>
+                <Menu.Item
+                  leftSection={<i className="bi bi-gear" />}
+                  component={Link}
+                  href="/team"
```

Comment:
this is a bit painful split to have but in local mode we don't have a team page to edit sources, maybe we should either keep the team page in local mode (a bit confusing?) or keep the modal here (more components to maintain just for local mode)

you can view the issue when trying to edit sources in the vercel preview of this PR

Reply:
oo good point. keeping the modal for local mode makes sense